operations/patch_util.go

```diff
@@ -676,6 +681,27 @@ func loadGitData(dir, branch, ref, commits string, format bool, extraArgs ...str
 	}, nil
 }
 
+func gitGetRemote(dir, owner, repo string) (string, error) {
```

Comment:
Can we please add a test for this function?

Reply:
or at least break out the regex manipulation bit to its own function and test that?